Neos.Flow/Configuration/Settings.Http.yaml

```diff
@@ -68,7 +68,7 @@ Neos:
               position: 'start'
               component: 'Neos\Flow\Session\Http\SessionResponseComponent'
             'flashMessages':
-              position: 'after setSessionCookie'
+              position: 'before setSessionCookie'
```

Comment:
We could make this `start` and `setSessionCookie: position: 'after flashMessages'` but I'm unsure on the potential effects. Considering this works and all components are supposed to be replaced by PSR-15 middlewares anyway, guess this is the safer approach.

Reply:
relative positioning (obviously) only works if the reference component is active, i.e. `after flashMessages` could behave inconsistent if the flashMessages component is disabled (I guess).
`start 10` and `start 20` could work as well, but I think it's fine the way it is now!